Assets/MRTK/Examples/Demos/Input/Scenes/InputActions/InputActions.MixedRealityControllerMappingProfile.asset

```diff
@@ -836,7 +836,7 @@ MonoBehaviour:
     - id: 2
       description: Grip Press
       axisType: 3
-      inputType: 13
+      inputType: 0
```

Comment:
Is this the correct value? I think this maps to `GripPress`, which isn't 0. Same for the other mapping profile. Might be worth trying to reserialize.